content/community/organization/governance.html

```diff
@@ -8,13 +8,39 @@
+
+<p>The development team is not a legal entity, but a group of people managing the technical
+aspects of the project: coding guidelines, tools to use for collaboration, and general
+direction of the development of Haiku.</p>
+
+<p>People are invited to become part of the development team after they have contributed
+sufficient work (for example in the form of patches, bug reports submissios, ...)
+and show a good understanding of the "Haiku Way" (the overall direction of the project
+and the way we collaborate). On the initiative of existing members of the team, a proposal is
+sent to the <a href="/community/ml">haiku-development mailing list</a> where the team members can veto it for a period
+of 15 days. If the proposal gets enough support and no one has objections, the membership
+becomes effective.</p>
+
+<p>The list of the team members and their contact addresses are listed <a href="https://review.haiku-os.org/admin/groups/3,members">on Gerrit</a>.</p>
+
 <h2>Haiku, Inc.</h2>
-<p>Haiku, Inc. Is a US based Not-for-Profit Organization that was founded in July of 2003 by the founder, long time leader, and visionary of the Haiku Project, Michael Phipps. Its aim is to support the Haiku Project and to further the development of the operating system and its online community.</p>
+<p><a href="https://haiku-inc.org">Haiku, Inc.</a> Is a US based Not-for-Profit Organization that was founded in July of 2003 by the founder, long time leader, and visionary of the Haiku Project, Michael Phipps. Its aim is to support the Haiku Project and to further the development of the operating system and its online community.</p>
 
-<p>Find out more about Haiku, Inc. in the <a href="/about/haiku_inc/">about us section</a> of this website.</p>
+<p>Haiku, Inc. is usually not involved in technical decisions and is doing mostly administrative work. Their main decision power is on allocating funds to various tasks such as development
+contracts or buying some hardware or software needed by developers for specific tasks.</p>
 
 <h2>Haiku Support Association (HSA)</h2>
 <p>The <a href="http://haiku-support-association.org/index-eng.html">Haiku Support Association</a> is a German based organisation of users and developers of Haiku. Their objective is to promote the development and further adoption of Haiku, which is done by focusing their efforts on organizing the regularly held BeGeistert event and its associated CodeSprint.</p>
 
 <h2>HaikuPorts</h2>
-<p><a href="https://github.com/haikuports/haikuports">HaikuPorts</a> is Team of Haiku users, developers and contributors who are cooperating towards porting applications and libraries to Haiku.</p>
\ No newline at end of file
+<p><a href="https://github.com/haikuports/haikuports">HaikuPorts</a> is Team of Haiku users, developers and contributors who are cooperating towards porting applications and libraries to Haiku.</p>
+
+<h2>HaikuArchives</h2>
```

Comment:
Not sure about this one.

```suggestion
 <h2>Haiku Archives</h2>
```

Reply:
-1, leaving as is